README.md

```diff
@@ -11,6 +11,23 @@ source ./venv/bin/activate
 pip install starttls-policy
 ```
 
+If you are getting the following error when installing via pip:
+
+```
+Collecting starttls-policy
+  Could not find a version that satisfies the requirement starttls-policy (from versions: )
+No matching distribution found for starttls-policy
+```
+
+then run the following instead, in order to get the module installed:
+
+```
+$ git clone https://github.com/EFForg/starttls-everywhere.git 
+$ cd starttls-everywhere/starttls-policy
+$ python setup.py install
+```
+
+
```

Comment:
I'm not sure this advice is still accurate for this repository. The things that have changed are:

 * The package and command has been renamed to `starttls-policy-cli`
 * `pip install .` (rather than `pip install starttls-policy`) should work just fine to install the tool properly.

The last thing is, we would prefer that people use a `venv` to test out the tool, rather than just running `python setup.py install`, to make sure different Python package sources (OS and pip) don't interfere with each other.